deployment/cfn/stacks.py

```diff
@@ -0,0 +1,41 @@
+from majorkirby import GlobalConfigNode
+
+from vpc import VPC
+from private_hosted_zone import PrivateHostedZone
+
+import ConfigParser
+
+
+def get_config(mmw_config_path, profile):
+    """Parses a configuration file
+
+    Arguments
+    :param mmw_config_path: Path to the config file
+    :param profile: Config profile to read
+    """
+    mmw_config = ConfigParser.ConfigParser()
+    mmw_config.optionxform = str
+    mmw_config.read(mmw_config_path)
+
+    return {k: v.strip('"').strip("'") for k, v in mmw_config.items(profile)}
```

Comment:
Consider adding some config validation here. `default.yml.example` may fall out of sync with reality over time.